modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties

```diff
@@ -22,6 +22,7 @@ allow-guest-users-to-send-files=Allow Guest Users to Send Files
 allow-multiple-selections=Allow Multiple Selections
 an-input-mask-helps-to-ensure-a-predefined-format=An input mask helps to ensure a predefined format. It can be useful for zip codes, card numbers, and phone numbers. For example, the mask +09 (999) 999-9999 becomes +12 (123) 123-1234 on a form.
 and=AND
+any-input-mask-character=Any input mask character
```

Comment:
@mateusgomes01 Brian prefers that all words begin with a capital letter if the sentence does not end with a period.
Example: `Any Input Mask Character`
That said, can you update the language keys? You can use rebase to drop/edit previous commits to make it easier. I can show you how to do it if you want.

Reply:
I was just following the mock-up. Should we change it?

Reply:
We don’t need to change the mockup, but Brian will probably want to change the code.

Reply:
Won't QA compare our developments with the mock-up? Won't that be flagged as a design inconsistency?

Reply:
We can talk to Victor, but we will not change the sentence so it is no a problem.

Reply:
Ok! I'll talk to him

Reply:
@carolmariaabb should the same rule be applied to the `Show more` / `Show less` button?

Reply:
Yes